### DISCUSSION TITLE
Rough day on the market

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#1241] [General]
Just some general rantings about the current stock market ...

**Comment by DaveSkender** [comment]
#### Days like this, make me think about technical indicators!

![chart](https://github.com/user-attachments/assets/e8a56292-248b-4ad1-a4d7-d0fd17f6be2f)

> Nasdaq 100 (QQQ) at 12pm ET on 08 Aug 2024, as shown in our demo site @ [charts.stockindicators.dev](https://charts.stockindicators.dev)

**Reply by LeeDongGeon1996** [reply]
Where are u going my TQQQ...

**Comment by bobtabor** [comment]
Wow, I haven't kept up with you in a while. Is that visualization part of your library? I just implemented a simple candlestick chart in WPF yesterday ... wish I would have known!

Hope you're doing well!

**Reply by DaveSkender** [reply]
Hi Bob, good to hear from you.  Those charts are a bit of a _couture hack_ that I'd built off of a charts.js base, specifically for that [demo site](https://charts.stockindicators.com).  It's not part of this library, nor is it in any shape for plug-n-play like an intentionally designed component, but [the site code](https://github.com/facioquo/stock-charts) is out there if you want to take a peek.  We also have a running list of [charting and visualization options](https://github.com/DaveSkender/Stock.Indicators/discussions/430) that people share.

**Reply by DaveSkender** [reply]
Someday I'll get around to taking some next steps to making these charts a bit more reusable and perhaps to incorporate more of the library's features.  Lately, I've been prioritizing work on the next major version of the library itself, to incorporate more _live data_ scenarios that better support [incremental and streaming use cases](https://github.com/DaveSkender/Stock.Indicators/discussions/1018).

**Comment by DaveSkender** [comment]
There are no indicators for this: U.S. Fed Chair used the words “_more cautious_”.

![](https://github.com/user-attachments/assets/ef7c2d48-eba7-45d1-82fb-427261013f93)

**Reply by bobtabor** [reply]
I anticipated this and sold off what I could before noon (but couldn't completely escape it). Tom Lee (Fundstrat, FSInsight, CNBC) was saying 2 cuts in 2025 and some in the market were saying 4 cuts. I tended to agree with Tom and knew there might be a sell off, but this got bloody. Hope you didn't get hit too hard, Dave!

**Reply by DaveSkender** [reply]
No major pain here, just trying to figure out the mechanics of it all.  These bots that auto re-level big funds go nuts on new data like this.  Instantly!  Hopefully it’s all done and doesn’t cascade much further.

**Reply by DaveSkender** [reply]
In retrospect, we are up ~20% since my last _rough day_ post.  Could go either way from here.

**Reply by GelLiNN** [reply]
@DaveSkender You're a legend man.  Our algo predictions are on hiatus after this event.  The froth is so strong that I can't in good conscience recommend bulls at the moment.  What worries me is that BTC was the defensive flee and that got git hard yesterday AND today.  Things seem fishy though I do believe attacks on BTC valuation are only temporary until about March.

**Reply by DaveSkender** [reply]
Crypto has a lot of momentum bots since it’s so open to algos — flash crash waiting to happen IMO.  But, looks like it’s also stabilizing with more bigger players like IBIT, MSTR, and others.
